sdk/helper/tlsutil/tlsutil.go

```diff
@@ -65,3 +67,37 @@ func GetCipherName(cipher uint16) (string, error) {
 	}
 	return "", fmt.Errorf("unsupported cipher %d", cipher)
 }
+
+func ClientTLSConfig(caCert []byte, clientCert []byte, clientKey []byte) (*tls.Config, error) {
+	var tlsConfig *tls.Config
+
+	switch {
+	case len(caCert) != 0 && len(clientCert) != 0 && len(clientKey) != 0:
+		// Valid
+	case len(caCert) != 0:
+		return nil, errors.New("ca cert, client key and client cert must all be set, or none should be set")
```

Comment:
Can you make this error a package var instead of having the same message multiple places? Then also we can check for the error by var.

Reply:
After Jim's comment, there is only one error message now, but still I created a var for this.